src/kansa-admin/components/Member.jsx

```diff
@@ -51,28 +51,39 @@ export const membershipTypes = [
   'Adult'
 ]
 
+const MemberTitle = ({ attr, member, ...props }) => {
+  const membership = member.get('membership')
+  const title = attr.member
+    ? `Member #${member.get('member_number')} (${membership})`
+    : /^DP/.test(membership)
+      ? membership.replace(/^DP/, 'Day pass:')
+      : 'Non-member'
```

Comment:
Complex decision logic going on here, I don't think this c style conditional works well in this context. Could we break this up into either longform if/else?

Reply:
I actually rather like the ternary operator, esp. with the Prettier-forced styling for it. There may be too many of these in the codebase to replace easily...